lib/helpers.rb

```diff
@@ -19,6 +19,14 @@ def comment
     @comment ||= Comment.find(params[:comment_id])
   end
 
+  def verify_or_fix_cached_comment_count(comment, comment_hash)
+    unless comment_hash["children"].nil?
+      if comment_hash["child_count"] != comment_hash["children"].length
+        comment.child_count = comment_hash["child_count"] = comment.update_cached_child_count(child_comments_count)
```

Comment:
I think the "comment.child_count = " is already handled by the update call.

This is what I had imagined.  Thanks.